From 948c16444f23da486a175c53031ab5259fb197cd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Sun, 10 Apr 2011 19:13:50 +0000
Subject: [PATCH] Makefile: extract Q_() source strings as ngettext()
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The patch adding the Q_() wrapper function around ngettext[1] didn't
contain a corresponding update to the "pot" target in the Makefile. As
a result "make pot" wouldn't extract gettext messages using the plural
form.

1. added by Jonathan Nieder in v1.7.5-rc0~54^2~2 (i18n: add stub Q_()
   wrapper for ngettext).

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index 92c1c5e..2441d56 100644
--- a/Makefile
+++ b/Makefile
@@ -2051,7 +2051,7 @@ XGETTEXT_FLAGS = \
 	--add-comments \
 	--msgid-bugs-address="Git Mailing List <git@vger.kernel.org>" \
 	--from-code=UTF-8
-XGETTEXT_FLAGS_C = $(XGETTEXT_FLAGS) --keyword=_ --keyword=N_ --language=C
+XGETTEXT_FLAGS_C = $(XGETTEXT_FLAGS) --keyword=_ --keyword=N_ --keyword="Q_:1,2" --language=C
 LOCALIZED_C := $(C_OBJ:o=c)
 
 po/git.pot: $(LOCALIZED_C)
-- 
1.7.4.1

